src/foam/nanos/client/Client.js

```diff
@@ -276,10 +276,12 @@ foam.CLASS({
         {
           name: 'cronDAO',
           factory: function() {
-            return this.EasyDAO.create({
-              daoType: 'CLIENT',
+            return this.RequestResponseClientDAO.create({
```

Comment:
Why is this change needed?

Reply:
I thought `RequestResponseClientDAO` was the correct DAO to connect the front-end `cronDAO` with the `cronDAO` nano in the back-end. Any suggestions?

Reply:
The old code looks correct.

Reply:
Does EasyDAO support http delegates? Like:

```
delegate: this.HTTPBox.create({
  method: 'POST',
  url: 'http://localhost:8080/cronDAO'
})
```

Accepting suggestions here.

Reply:
@diaswrd All you should be required to do is:
```
return this.EasyDAO.create({
  daoType: 'CLIENT',
  of: this.Cron,
  serviceName: 'cronDAO'
});
```

Was that not working?

Reply:
@eatonk it was working, but the changes on the GUI were not being persisted in the cronDAO service running on the server. A page reload would wipe any new cronjobs created.